src/test/lib/sql/sql_translator_test.cpp

```diff
@@ -3132,6 +3136,22 @@ TEST_F(SQLTranslatorTest, CastStatement) {
     const auto [actual_lqp, translation_info] = sql_to_lqp_helper("SELECT CAST('2000-01-01' as DATE);");
     EXPECT_LQP_EQ(actual_lqp, expected_lqp);
   }
+  {
+    // Date times need to be resolved, as we do not have a DateTime DataType
```

Comment:
What does "resolved" mean here?

Reply:
`resolved/translated to a value expression`?